src/scss/components/topnav/index.scss

```diff
@@ -159,30 +162,36 @@
   #topnavProductDropdown,   // deprecated
   .hxTopNavMenu > hx-menu {
```

Comment:
Please validate `<hx-menu>`, `<hx-menuitem>`, `.hxMenuKey`, `.hxMenuValue`, header/footer, etc. against the new specs.   It looks like some of the font values and possibly padding/margins have changed.

Reply:
i have validated and done the changes. please review